printing/printer.ml

```diff
@@ -34,6 +34,49 @@ let delayed_emacs_cmd s =
 let get_current_context () =
   Pfedit.get_current_context ()
 
+let enable_unfocussed_goal_printing = ref false
+let enable_goal_tags_printing = ref false
+let enable_goal_names_printing = ref false
+
+let should_tag() = !enable_goal_tags_printing
+let should_unfoc() = !enable_unfocussed_goal_printing
+let should_gname() = !enable_unfocussed_goal_printing
+
+
+let _ =
+  let open Goptions in
+  declare_bool_option
+    { optsync  = false;
```

Comment:
The usual issue: do we want the command to be possibly part of a document or not, and hence synchronized with backtracking?

The usage seems that the `Printing` commands are `sync`, so by consistency, until a protocol is found to deal with options usable both as a command and from a menu, I would lean towards making the option `sync`.